testing/cli-e2etest/Makefile

```diff
@@ -17,6 +18,14 @@ test: # run tests for this application
 	go clean -testcache; \
 	go test -v -timeout 300s -p 1 ./...;
 
+test/scenario: # run tests for this application
```

Comment:
This will allows us execute a single testscenario

Reply:
Good idea! 🎉